docs/topics/cache.txt

```diff
@@ -165,6 +163,12 @@ particularly temporary.
 
     The ``PyMemcacheCache`` backend was added.
 
+.. deprecated:: 3.2
+
+    The ``MemcachedCache`` backend is deprecated as it has some problems and
+    seems to be unmaintained. Use ``PyLibMCCache`` or ``PyMemcacheCache``
+    instead.
```

Comment:
See my comment made against the change in the release notes about backend vs client library.